DockerFileTests

```diff
@@ -1,17 +1,8 @@
-FROM python:latest
-ARG GCLOUD_KEY
-ARG PROJECT_NAME
-ARG CLUSTER_NAME
-ARG CLUSTER_ZONE
-
-COPY . .
-
-ADD  . /go-spacemesh
+FROM python:latest AS tests_base
 
 WORKDIR /go-spacemesh/tests
 
 RUN pip install pytest
```

Comment:
Maybe it is better to have the pytest in different layer from kubectl

Reply:
Why?

I mean, everything up to `tests_base` will be cached and remain constant unless the dockerfile is changed, since its all static and not dependant on any other files in the repository.

I could still see the value if you'd use one of them in another dockerfile or something. Is that the case?

Reply:
Or unlesas pytest is changed. (Maybe it is better to add pytest to the requirements file with specific version)

Reply:
Agree about specific version 👍 

Let's do it in a future PR.